tardis/model/tests/test_csvy_model.py

```diff
@@ -8,24 +8,32 @@
 from tardis.model import Radial1DModel
 import pytest
 
-DATA_PATH = os.path.join(tardis.__path__[0],'model','tests','data')
-
-@pytest.fixture(scope="module", params=[('config_csvy_full.yml','config_csvy_full_old.yml'),
-                                        ('config_csvy_nocsv_branch85.yml','config_csvy_nocsv_branch85_old.yml'),
-                                        ('config_csvy_nocsv_uniform.yml','config_csvy_nocsv_uniform_old.yml'),
-                                        ('config_csvy_nocsv_powerlaw.yml','config_csvy_nocsv_powerlaw_old.yml'),
-                                        ('config_csvy_nocsv_exponential.yml','config_csvy_nocsv_exponential_old.yml'),
-                                        ('config_csvy_full_rad.yml','config_csvy_full_rad_old.yml')])
-def filename(request):
-    fn_tup = request.param
-    fn = os.path.join(DATA_PATH, fn_tup[0])
-    fnold = os.path.join(DATA_PATH, fn_tup[1])
-    return fn, fnold
-
-def test_compare_models(filename):
-    fn, fnold = filename
-    tardis_config = Configuration.from_yaml(fn)
-    tardis_config_old = Configuration.from_yaml(fnold)
+DATA_PATH = os.path.join(tardis.__path__[0], "model", "tests", "data")
+
+
+@pytest.fixture(
+    scope="module",
+    params=[
+        "model_full",
+        "branch85",
+        "uniform",
+        "powerlaw",
+        "exponential",
+        "radiative",
+    ],
+)
+def model_config_files(request):
```

Comment:
add brief docstring.

Reply:
and I would say - call if `model_config_fnames`